packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -80,39 +94,85 @@ const balanceOptions = [
   { name: "Max", divider: BalanceDivider.MAX },
 ];
 
+const customParseAsTimestamp = createParser({
+  parse: (v) => {
+    const ms = parseInt(v);
+    if (Number.isNaN(ms)) {
+      return null;
+    }
```

Comment:
what happens if it returns null? 

Shouldn't it return now if doesn't have start time?

Reply:
If it returns `null` it will use the default value. This was copied from the library so our custom `parseAsTimestamp`